index.ts

```diff
@@ -174,15 +179,16 @@ LabelSyncer.syncLabels(
                     case "labeled":
                     case "unlabeled":
                         // Find issue number from target repo where the issue title matches the title of the issue in the source repo
-                        octokit.request('GET /repos/{owner}/{repo}/issues', {
```

Comment:
same approach needs to be implemented here: https://github.com/camunda/issue-sync-action/blob/main/issueSyncer.ts#L6

Reply:
maybe even introduce a reusable method `getIssueByTitle` for both cases.

Reply:
that's why I mentioned if the og maintainer abandoned it, it may makes sense to refactor the whole thing.

Reply:
The issueSyncer didn't have any actual issue syncing logic, therefore renamed it to `github` and `GitHub` class, which we can longterm refactor everything to.

Adjusted the function `getIssueNumberByTitle` to include the change and changed the existing call to reuse this function.